### DISCUSSION TITLE
Validate dynamic multi-select boxes

### DISCUSSION THREAD

**Discussion by Uzeyir9695** [primefaces/community#710] [PrimeVue]
In the documentation there is given example of the validation for only one Multi-select box.
What if I have dynamic multi-select boxes or input fields, how can be validated these?
And how to get dynamic multi-select boxes values, for example if I have 15 select-box or input fields?

**Comment by tugcekucukoglu** [comment]
If you could provide an example, we could help. Without a reproducer demo, it's hard to guess the problem.

**Reply by Uzeyir9695** [reply]
So, below PrimeSelect box is dynamic. In this case how can I validate each of them and display error messages under for each box?
Below code generates select boxes as it looks on the screenshot attached.


`<PrimeSelect
               v-for="filter in carsFilters"
               v-model="selectedFilters[filter.name]"
               filter
               @change="selectFilter"
               :id="filter.name"
               :options="[filter]"
               optionGroupLabel="name"
               optionGroupChildren="values"
               :placeholder="filter.name"
               :selectionLimit="1"
               class="w-full sm:w-19rem mt-3"
               panelClass="z-1 select-none"
           />`
           
![filters](https://github.com/primefaces/community/assets/74812496/5471abea-4705-48e0-96fb-83dbfee938ac)

**Comment by Uzeyir9695** [comment]
I am waiting for a response. Please answer to my question.
